src/api/app/views/webui/packages/build_log/_breadcrumb_items.html.haml

```diff
@@ -1,5 +1,10 @@
 %li.breadcrumb-item.text-word-break-all
+  %i.fa.fa-cubes
+  = link_to(@project, project_show_path(@project))
   %i.fa.fa-archive
```

Comment:
@hennevogel the package part of the breadcrumbs needs to be nested under another `%li.breadcrumb-item` otherwise it looks off

Reply:
better?

Reply:
Yes :)